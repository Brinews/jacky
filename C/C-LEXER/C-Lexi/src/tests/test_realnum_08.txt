


      2;3


0.0000200
