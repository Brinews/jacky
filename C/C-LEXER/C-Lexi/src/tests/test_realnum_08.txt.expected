{2 , NUM , 4}
{ , SEMICOLON , 4}
{3 , NUM , 4}
{0.0000200 , REALNUM , 7}
{ , END_OF_FILE , 9}
